### ISSUE TITLE
Papis not adding a ref field to any documents

### ISSUE THREAD

**Issue by JBlocklove** [papis/papis#254]
When I add a document to a library now no `ref:` field is added. This seems to create a conflict with `papis-vim`, since it gets the references through `{doc[ref]}`.

I am currently up to date with the `master` branch.

**Comment by avonmoll** [comment]
I noticed this as well when adding things via doi (which I think, in turn, utilizes the CrossRef importer stuff in `crossref.py`). I added the following right after [this line](https://github.com/papis/papis/blob/b2de2520bfb57447d6f5343a95ab7aec37505dab/papis/crossref.py#L158) to my local install and now refs are generated and added.

```python
new_data['ref'] = papis.config.get("ref-format").format(doc=new_data)
```

To me, this is a sensible place to have the logic. As a side note, there seems to be some inconsistency in the code base on using `doc` in formatting strings (since this is actually a configuration option `format-doc-name`, whose default is `doc`).

Are you experiencing missing `ref` with an importer other than CrossRef?
